torch/jit/annotations.py

```diff
@@ -317,7 +317,8 @@ def try_ann_to_type(ann, loc):
         if not is_enum_support_enabled():
             warnings.warn("Enum support is work in progress, enum class {}"
                           " is not compiled".format(ann))
-            return None
+        if not hasattr(ann, "__torch_script_class__"):
+            torch.jit._script._recursive_compile_class(ann, loc)
```

Comment:
Dumb question but how does this work? I thought the Python frontend only parses out methods from class definitions and the compiler relies on `__init__` to figure out what attributes the class has?